crates/cli/commands/src/stage/run.rs

```diff
@@ -130,6 +131,7 @@ impl<C: ChainSpecParser<ChainSpec = ChainSpec>> Command<C> {
                     target_triple: VERGEN_CARGO_TARGET_TRIPLE,
                     build_profile: BUILD_PROFILE_NAME,
                 },
+                ChainSpecInfo { name: format!("{}", provider_factory.chain_spec().chain) },
```

Comment:
you can just do `to_string` here

Reply:
Fixed in https://github.com/paradigmxyz/reth/pull/10737/commits/e8a877133a58ff5730ebbcc61cba5dd784eb118b